### PR TITLE
Android update

### PR DESCRIPTION
**Pull request by m1ga**
* update library

### CODE REVIEW

**Review by hansemannn — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
 dependencies {
-	implementation 'com.google.firebase:firebase-analytics:19.0.2'
-	implementation 'com.google.firebase:firebase-crashlytics:18.2.4'
+	implementation 'com.google.firebase:firebase-analytics:21.0.0'
```
Comment (hansemannn) [inline android/build.gradle:2]: We use 19.0.0 in https://github.com/hansemannn/titanium-firebase-analytics/blob/main/android/build.gradle#L6 right now. Couldn't this clash? Trying to be very cautious here!

Comment (m1ga) [reply android/build.gradle:2]: firebase-analytics:21.0.0 + firebase-crashlytics:18.2.10 are in the same "Android BoM" (30.0.1) so they should work together. The analytics update is here: https://github.com/hansemannn/titanium-firebase-analytics/pull/71

Comment (hansemannn) [reply android/build.gradle:2]: Ahh okay. I don't get the BoM versioning so far, but then we're good. I'll go over the Firebase modules over the weekend as well

Comment (m1ga) [reply android/build.gradle:2]: I only use [this link](https://firebase.google.com/support/release-notes/android) to get the latest versions :rofl: 
But their "old" version numbers scare me from time to time too. Not sure why they not just recompile the old stuff like `com.google.firebase:firebase-appcheck | 16.0.0` with "no changes" in order to keep the version number in sync.

**Pull request merged**